tests/pfcwd/conftest.py

```diff
@@ -244,3 +244,28 @@ def pfcwd_pause_service(ptfhost):
         needs_resume["garp_service"] = False
 
     logger.debug("pause_service needs_resume {}".format(needs_resume))
+
+
+@pytest.fixture(scope="function", autouse=False)
+def skip_pfcwd_higher_speeds(
+        duthosts,
+        enum_rand_one_per_hwsku_frontend_hostname,
+        setup_pfc_test):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if duthost.facts["asic_type"] != "cisco-8000":
+        yield
+        return
+
+    test_ports = setup_pfc_test['test_ports']
+    for test_port in test_ports:
+        speed = duthost.show_and_parse(
+            "show interfaces status {}".format(test_port))[0]['speed']
+        speed_int = speed.split("G")[0]
+        if int(speed_int) > 100:
+            pytest.xfail(
```

Comment:
Hi @kevinskwang , please take a look at this logic.
Dut to the 400G port/fanout can't be stably trigger the pfcwd, we plan to xfail it on Cisco 8000 platforms.
For now the logic covers T0/T1/T2,
Please let me know if you want to keep the old logic of T0/T1 devices, we can update it to cover T2 only